src/main/java/org/opentripplanner/street/model/vertex/StreetVertex.java

```diff
@@ -19,7 +22,7 @@
 public abstract class StreetVertex extends Vertex {
 
   /** All locations for flex transit, which this vertex is part of */
-  public Set<AreaStop> areaStops;
+  private Set<AreaStop> areaStops;
```

Comment:
Related to https://github.com/opentripplanner/OpenTripPlanner/pull/5233/files#r1263815874, would it make more sense to initialize this with an empty set?

Reply:
Yes, that is a good point. I have rewritten that code and made easier to read without consuming more memory.